apps/frontend/app/entry.server.tsx

```
@@ -1,38 +1,40 @@
 ) {
+  let shellRendered = false;
+  const userAgent = request.headers.get("user-agent");
+
   const body = await renderToReadableStream(
-    <RemixServer context={remixContext} url={request.url} />,
+    <ServerRouter context={routerContext} url={request.url} />,
     {
       onError(error: unknown) {
-        // Log streaming rendering errors from inside the shell
-        console.error(error);
         responseStatusCode = 500;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)